src/content/guides/payment-terminals.mdx

```diff
@@ -0,0 +1,78 @@
+
+> You are responsible for the safety of your API keys
+
+## Merchant Configs
+Centrapay [Merchant Configs](https://docs.centrapay.com/api/merchant-configs) represent an available set of configured payment methods that can be utilized by one or more payment terminals by a Merchant.
+
+Merchants and merchant configs are managed by Centrapay. The only details you need to be able to create payment requests on their behalf are the `merchantId` and `merchantConfigId` (aka `clientId`). Centrapay will send you these details through whatever channel works for you. Please get in touch to let us know.
+
+In the future Centrapay will allow an integrator to on-board their own merchants through APIs using integrator API keys.
+
+## Example Flows
+
+See [Payment Flows](/guides/payment-flows) for an overview of the API calls required for different payment flows.
+
+## Terminal Interface Guidelines
+
+When configuring a terminal with Centrapay there are a few common touch points that require branded assets. Please use the [Centrapay Brand Assets](https://centrapay.com/brand-assets/) when building the UI for these screens.
+
+### Displaying a Button
+
+![illustration of payment terminal displaying Centrapay button](/terminals/display-centrapay-button.png)
+
+Touch displays should show a button that loads the QR Code screen when pressed.
+
+### Displaying an Icon
+
+![illustration of payment terminal displaying Centrapay icon](/terminals/display-centrapay-icon.png)
+
+Displays that don’t support touch should show an icon above the button used to load the QR Code screen.
+
+### Displaying QR Codes
+
+![illustration of payment terminal displaying Centrapay QR code](/terminals/displaying-centrapay-qr-code.png)
+
+Centrapay QR codes should be displayed with a Centrapay logo in the centre.
+
+## Integration Architecture
+
+We strongly recommend Centrapay APIs are invoked from your backend and not directly from your payment terminals. Centralizing the invocation of our APIs from your backend offers the following benefits:
+
+1. Merchant network administrators will not need to make additional firewall changes.
+2. Your Centrapay API Keys can be managed centrally inside your secured network.
+3. TLS negotiation with Centrapay APIs can be managed independently of terminal hardware and software updates.
+
+## Mitigating Network Issues
+
+The following mitigations will be helpful to deal with inconsistencies in Payment Request statuses due to network issues and race conditions:
+
+**Respect Payment Request Status**
+
+Use the Payment Request status as the source of truth when determining if a Payment Request is paid or expired. For example, if cancelling a Payment Request fails with a `REQUEST_PAID` error then terminal should respond accordingly; either show transaction as paid or initiate a void instead.
```

Comment:
If you want extra points you could link the mention of void to the void endpoint docs but up to you.

Reply:
```suggestion
Use the Payment Request status as the source of truth when determining if a Payment Request is paid or expired. For example, if cancelling a Payment Request fails with a `REQUEST_PAID` error then terminal should respond accordingly; either show transaction as paid or initiate a [Void](https://docs.centrapay.com/api/payment-requests#void-a-payment-request) instead.
```